Scripts/Biochemistry/Rebalance_Reactions.py

```diff
@@ -11,22 +11,25 @@
 
 Update_Reactions=0
 for rxn in sorted(Reactions_Dict.keys()):
+#    if(rxn != "rxn00013"):
+#        continue
```

Comment:
This code seems to duplicate logic in Adjust_Reaction_Protons.py. Can we consolidate it?

Reply:
No, it started off like that, but it got tricky. There is an internal self-reference between status and stoichiometry when attempting to add/remove compounds, and it got more complex when considering whether users would add new compounds with an incorrectly stated number of protons. It was easier for me to separate it out into two scripts.